utils/check_repo.py

```diff
@@ -75,6 +78,9 @@
 # should **not** be the rule.
 IGNORE_NON_AUTO_CONFIGURED = [
     # models to ignore for model xxx mapping
+    "Speech2TextTransformerEncoder",
+    "Speech2TextTransformerDecoder",
+    "Speech2TextTransformerDecoderWrapper",
```

Comment:
Think those classes: `Speech2TextTransformerDecoderWrapper` can be deleted for now